### DISCUSSION TITLE
What should I do if I want to have a continuous conversation?

### DISCUSSION THREAD

**Discussion by AA8j** [acheong08/ChatGPT#1037] [Q&A] [answered]
How should I set it in python so that every conversation will not be a new window?

**Comment by wzqstudio** [comment]
want to know too

**Comment by wzqstudio** [comment]
@acheong08

**Comment by acheong08** [comment, marked as answer]
In V3, save your conversation to a file and load it again when you run the next time

For V1, use conversation_id
